app/controllers/donations_controller.rb

```diff
@@ -24,12 +24,12 @@ def create
     respond_to do |format|
       if @donation.save
         DonationMailer.thankyou(@donation).deliver_now
-        format.html { redirect_to root_url, 
+        format.html { redirect_to root_url,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.